GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -45,43 +45,60 @@ private void RevisionInfoLinkClicked(object sender, LinkClickedEventArgs e)
             try
             {
                 var url = e.LinkText;
-                var data = url.Split(new[] { '#' }, 2);
-
-                try
+                var count = url.Count(c => c == '#');
+                if (count == 1)
```

Comment:
what if there is no '#'? you would be unnecessary executing `else` clause code, wouldn't you?